packages/unit-testing/README.md

```diff
@@ -12,17 +12,15 @@ Suitecloud Unit Testing allows you to use unit testing with [Jest](https://jestj
 
 ## Features
 - Provides a default configuration to run unit tests with Jest in SuiteCloud projects.
-- Supports unit testing for SuiteScript 2.X files.
-- Provides SuiteScript 2.X stubs for the following SuiteScript modules:
-  - **N/http**
-  - **N/https**
-  - **N/record**
-  - **N/search**
-- Allows you to create custom stubs for SuiteScript 2.X or custom modules.
-- Supports code coverage capabilities through Jest.
+- Supports unit testing for SuiteScript 2.x files.
+- Provides SuiteScript 2.x stubs for SuiteScript modules.
+- Allows you to create custom stubs for SuiteScript 2.x or custom modules.
+
+For more information about the available SuitScript 2.x modules, see [SuiteScript 2.x Modules](https://nlcorp.app.netsuite.com/app/help/helpcenter.nl?fid=chapter_4220488571.html).  
+For more information about all the mockable stubs, see the CORE_STUBS list in the [Stubs](https://github.com/oracle/netsuite-suitecloud-sdk/tree/master/packages/unit-testing/stubs) folder.
```

Comment:
CORE_STUBS is an array that is defined inside `packages\unit-testing\jest-configuration\SuiteCloudJestConfiguration.js`, each element of the array includes the name we give to the mocked module, and the file it's referencing inside `packages/unit-testing/stubs`.
If we make reference to CORE_STUBS the link should be to [SuiteCloudJestConfiguration.js](https://github.com/oracle/netsuite-suitecloud-sdk/tree/masterpackages\unit-testing\jest-configuration\SuiteCloudJestConfiguration.js) file.

Reply:
I also think that the link can be relative right: [SuiteCloudJestConfiguration.js](./jest-configuration\SuiteCloudJestConfiguration.js) 

```
as maybe the comment above is formatted by markdown,
here it is the same content inside a code block:
 [SuiteCloudJestConfiguration.js](./jest-configuration\SuiteCloudJestConfiguration.js) 
```

Reply:
Updated!